docs/attributes-registry/artifact.md

```diff
@@ -0,0 +1,18 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Artifact
+
+## Artifact Attributes
+
+This group describes attributes specific to artifacts.
+
+| Attribute          | Type   | Description                                                                                                                                                                                     | Examples                                                                                                  | Stability                                                        |
+| ------------------ | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `artifact.hash`    | string | The full [hash value (see glossary)](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-5.pdf), often found in checksum.txt on a release of the artifact and used to verify pakage integrity. | `9ff4c52759e2c4ac70b7d517bc7fcdc1cda631ca0045271ddd1b192544f8a3e9`                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `artifact.name`    | string | The human readable file name of the artifact, often generated by build and release utilities to include the package name and version.                                                           | `golang-binary-amd64-v0.1.0`; `docker-image-amd64-v0.1.0`; `release-1.tar.gz`; `file-name-package.tar.gz` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
maybe related: https://github.com/open-telemetry/semantic-conventions/issues/1229
(adding for future reference)